rxv/rxv.py

```diff
@@ -124,6 +125,30 @@ def on(self, state):
     def off(self):
         return self.on(False)
 
+    def play(self):
+        self._send_action('Play')
+
+    def pause(self):
+        self._send_action('Pause')
+
+    def stop(self):
+        self._send_action('Stop')
+
+    def next(self):
+        self._send_action('Skip Fwd')
+
+    def previous(self):
+        self._send_action('Skip Rev')
+
+    def _send_action(self, action):
+        src_name = self._src_name(self.input)
+        if not src_name:
+            return None
```

Comment:
I was thinking about the same thing but afaik the only way to know if playback control is available is to check the input source. So we would have to compare the current input source with a list of pre-defined sources that we know support this. Not pretty solution but I guess that could work as long as we can find a complete list.